src/collapse/mod.rs

```diff
@@ -1,22 +1,46 @@
-mod bpftrace;
-mod perf;
+//! Tools for collapsing the output of various profilers (e.g. [bpftrace], [perf]) into the
+//! format expected by flamegraph.
+//!
+//! [bpftrace]: https://github.com/iovisor/bpftrace
+//! [perf]: https://en.wikipedia.org/wiki/Perf_(Linux)
 
-pub use bpftrace::Bpftrace;
-pub use perf::{Perf, PerfOptions};
+pub mod bpftrace;
+pub mod perf;
 
 use std::fs::File;
 use std::io;
 use std::path::Path;
 
 const READER_CAPACITY: usize = 128 * 1024;
 
+/// Trait representing the ability to collapse the output of various profilers, such as [bpftrace]
```

Comment:
Let's make this more active-sounding: "This trait represents ..."

Reply:
👍